app/controllers/katello/api/v2/host_packages_controller.rb

```diff
@@ -91,6 +93,11 @@ def resource_class
 
     private
 
+    def latest_upgradables
```

Comment:
Curious about this approach of getting these packages through a controller vs the underlying ruby code? This is a new one for me!

Reply:
Very interesting. Wouldn't the following achieve the same?
``` ruby
    def latest_upgradables
      ::Katello::Rpm.installable_for_hosts([@host])
    end   
```